package.json

```diff
@@ -29,7 +29,7 @@
     "bcrypt-nodejs": "0.0.3",
     "bitbucket-api-v2": "^0.6.2",
     "bitbucket-v2": "^0.4.3",
-    "bluebird": "^3.5.1",
+    "bluebird": "^3.5.3",
```

Comment:
Maybe is risky to update this on this Pull Request, and that's why a new `package-lock.json` was generated